tests/test_remote.py

```diff
@@ -42,12 +43,19 @@ def setUpModule():   # pylint: disable=invalid-name
     bootstrap.setup_component(
         hass, http.DOMAIN,
         {http.DOMAIN: {http.CONF_API_PASSWORD: API_PASSWORD,
-         http.CONF_SERVER_PORT: MASTER_PORT}})
+                       http.CONF_SERVER_PORT: MASTER_PORT}})
 
     bootstrap.setup_component(hass, 'api')
 
-    hass.start()
-    time.sleep(0.05)
+    def block_start(hass_obj):
```

Comment:
You are replicating exactly what `hass_obj.start()` does: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/core.py#L127-L128

Sleeping 0.05 seconds allows Python switch to the WSGI server thread and boot up the sockets.

[Read more about the GIL and how and why it works](https://wiki.python.org/moin/GlobalInterpreterLock)